app/signer/bls.go

```diff
@@ -81,24 +80,21 @@ func GenBls(keyFilePath, passwordFilePath, password string) *Bls {
 	return pv
 }
 
-// LoadBls returns a Bls after loading the erc2335 type of structure
-// from the file and decrypt it using a password.
+// LoadBls loads a BLS key, attempting to use environment variable first and file-based passwords after.
 func LoadBls(keyFilePath, passwordFilePath string) *Bls {
```

Comment:
If we allow nil `Bls` returned, then we'd better rename it to something like `TryLoadBlsFromFile` and add a note that nil `Bls` could be returned due to non-existence of the key file